ci/gpu/build.sh

```diff
@@ -112,6 +112,8 @@ cmake ..
 make -j${PARALLEL_LEVEL}
 cd ../python
 python setup.py install
+cd ../runtime/python
+python setup.py install
```

Comment:
This is why Treelite and its runtime should be released as Conda packages.

Reply:
There is an attempt at a conda-forge package in PR ( https://github.com/conda-forge/staged-recipes/pull/11338 ). It looks like there are some reviewer comments that need to be addressed, but nothing too difficult. We could resubmit as a new PR with those comments addressed.

Reply:
While we await conda packages, will every developer need to run this command? or only some developers sometimes?

Reply:
No, only those users who want to opt into a few optional features that require the Treelite python package, such as importing scikit-learn models.

Reply:
I had an offline discussion with @JohnZed. The lack of Treelite package on Conda and PyPI (*) is a deal-breaker, since features like scikit-learn model import depend on the Python Treelite package. We've decided to delay this PR to 0.15 release, so that I will have time to release latest Treelite to Conda and PyPI.